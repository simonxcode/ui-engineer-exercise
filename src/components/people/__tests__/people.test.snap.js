import React from 'react'
import { shallow } from 'enzyme'
import People from '../people'

describe('Component - People', () => {
  let props
  
  it('should match snapshot', () => {
    const wrapper = shallow(<People {...props }/>)
    expect(wrapper).toMatchSnapshot()
  })
})
