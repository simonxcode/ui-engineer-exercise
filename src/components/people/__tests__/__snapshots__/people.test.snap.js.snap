// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - People should match snapshot 1`] = `ShallowWrapper {}`;
